SpatialGDK/Source/SpatialGDKTests/Private/RPCContainer/RPCContainerTest.cpp

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDefinitions.h"
+
+#include "ObjectDummy.h"
+#include "ObjectSpy.h"
+
+#include "Utils/RPCContainer.h"
+#include "Schema/RPCPayload.h"
+
+#include <Core.h>
+
+#define RPCCONTAINER_TEST(TestName) \
+	TEST(FRPCContainer, TestName)
+
+using namespace SpatialGDK;
```

Comment:
Why not put this in that namespace nested with another test namespace?

Reply:
Currently it looks like these are all globals

Reply:
Is there a problem with that?

Reply:
I'd be surprised if something broke but it's not great to have global stuff kicking around.
In this case it's also part of the SpatialGDK so it seems like that stuff should be in that namespace.

Reply:
It's in SpatialGDKTests module, so it's not a part of SpatialGDK.

Reply:
The namespace can exist across modules. The tests are part of the gdk too. Which is why I'd normally put things in nested namespaces.

Reply:
What do you suggest to put in a namespace?